content/en/docs/tasks/administer-cluster/change-pv-access-mode-readwriteoncepod.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Change the Access Mode of a PersistentVolume to ReadWriteOncePod
+content_type: task
+weight: 90
+---
+
+<!-- overview -->
+This page shows how to change the access mode on an existing PersistentVolume to
+use ReadWriteOncePod.
+
+## {{% heading "prerequisites" %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+{{< note >}}
+The ReadWriteOncePod access mode is only supported for CSI volumes and
+Kubernetes version 1.22+. To use this this feature you will need to update the
+following [CSI sidecars](https://kubernetes-csi.github.io/docs/sidecar-containers.html)
+to these versions or greater:
+
+* [csi-provisioner:v3.0.0+](https://github.com/kubernetes-csi/external-provisioner/releases/tag/v3.0.0)
+* [csi-attacher:v3.3.0+](https://github.com/kubernetes-csi/external-attacher/releases/tag/v3.3.0)
+* [csi-resizer:v1.3.0+](https://github.com/kubernetes-csi/external-resizer/releases/tag/v1.3.0)
+{{< /note >}}
+
+<!-- steps -->
+
+## Why should I use ReadWriteOncePod?
+
+Prior to Kubernetes 1.22, the ReadWriteOnce access mode was commonly used to
+restrict PersistentVolume access for workloads that required single-writer
+access to storage. However, this access mode had a limitation: it restricted
+volume access to a single *node*, allowing multiple pods on the same node to
+read from and write to the same volume simultaneously. This could pose a
+significant risk for applications that demand strict single-writer access for
+data safety.
+
+If ensuring single-writer access is critical for your workloads, consider
+migrating your volumes to ReadWriteOncePod.
+
+## Migrating existing PersistentVolumes
+
+If you have existing PersistentVolumes, they can be migrated to use
+ReadWriteOncePod.
+
+In this example, we already have a "cat-pictures-pvc" PersistentVolumeClaim that
+is bound to a "cat-pictures-pv" PersistentVolume, and a "cat-pictures-writer"
+Deployment that uses this PersistentVolumeClaim.
```

Comment:
Can you clarify that the existing access mode on the PV is ReadWriteOnce?
Also can you clarify that whether migrating from ReadWriteOnce to ReadWriteOncePod is the only migration path, meaning you can't migrate from any other access mode to RWOP?

Reply:
Updated to clarify ReadWriteOnce volume. Also noted that migrations are supported from all access modes to ReadWriteOncePod (verified by running steps for all other access modes).